SecondAssignmentFirstProgram.cs

```diff
@@ -0,0 +1,178 @@
+                    {
+                        pos = partoneone.IndexOf('?');
+
+                        string newpart = partoneone.Remove(pos, 1);
+                        FirstNewNumber = int.Parse(partonetwo);
+                        SecondNewNumber = int.Parse(newpart);
+
+                     
+
+                            int NewResult = FirstNewNumber * SecondNewNumber;
+
+                            if (parttwo.Equals(NewResult.ToString()))
+                            {
+                                missingDigit = -1;
+                            }
+                            else
+                            {
+                                int second = int.Parse(parttwo) / FirstNewNumber;
+
+                                string newresult = second.ToString();
+
+                                char digit = newresult[pos];
+
+                                missingDigit = int.Parse(digit.ToString());
+
+                                
+                            }
+
+                           
+                    }
+                    else
+                    {
+                        missingDigit = -1;
+                    }
+
+
+
+
+                }
+                else if(partonetwo.Contains('?'))
+                {
+                 
+                    if(int.Parse(parttwo) % int.Parse(partoneone) == 0)
+                    {
+                        pos = partonetwo.IndexOf('?');
+
+
+                        string newpart = partonetwo.Remove(pos, 1);
+                        FirstNewNumber = int.Parse(partoneone);
+                        SecondNewNumber = int.Parse(newpart);
+
+                        NewResult = FirstNewNumber * SecondNewNumber;
+
+                            if (parttwo.Equals(NewResult.ToString()))
+                            {
+                                missingDigit = -1;
+                            }
+                            else
+                            {
+                                int second = int.Parse(parttwo) / FirstNewNumber;
+
+                                string newresult = second.ToString();
+
+                                char digit = newresult[pos];
+
+                                missingDigit = int.Parse(digit.ToString());
+
+                            }
+
+                    }
+                    else
+                    {
+                        missingDigit = -1;
+                    }
+                    
+
+                }
+                
+                
+
+            }
+            else if(parttwo.Contains('?'))
+            {
+                pos = parttwo.IndexOf('?');
+                
+                digitsArray = partone.Split('*');
+
+                FirstNumber = int.Parse(digitsArray[0]);
+
+                SecondNumber = int.Parse(digitsArray[1]);
+
+                ResultNumber = FirstNumber * SecondNumber;
+
+                string result = ResultNumber.ToString();

```

Comment:
Here int is converted to string and again below string is converted to int. Can you please check this.